nilearn/decoding/searchlight.py

```diff
@@ -249,6 +250,22 @@ class SearchLight(BaseEstimator):
         when y is supplied.
     %(verbose0)s
 
+    Attributes
+    ----------
+    scores_ : numpy.ndarray
+        3D array containing searchlight scores for each voxel, aligned
+         with the mask.
+
+         .. versionadded:: 0.11.0
```

Comment:
Can you add that for all the new attributes?

Reply:
Were then added in version 0.9.0 (process_mask_ and masked_scores_)